### ISSUE TITLE
[Accessibility][Form and validations] 'Click to Add a person' label is not associated with the first contact list item.

### ISSUE THREAD

**Issue by joschect** [microsoft/fluentui#11020]
Breakout of #6259

**Comment by joschect** [comment]
Only an issue with the controlled picker example
